src/elm/Page/Shop/Editor.elm

```diff
@@ -356,15 +357,15 @@ viewForm shared balances imageStatus isEdit isDisabled deleteModal form =
                         , disabled isDisabled
                         ]
                         []
+                    , viewFieldErrors (listErrors shared.translations form.description)
                     ]
                 , div [ class "mt-2" ]
                     [ formField
                         [ label
                             [ class "input-label" ]
                             [ text (t "shop.which_community_label") ]
-                        , viewFieldErrors (listErrors shared.translations form.symbol)
                         , select
-                            [ class "input w-full mb-2 form-select select"
+                            [ class "input w-full form-select select"
```

Comment:
We don't need `input` class for the select element. `.input` is for inputs and textareas. I know that this class was there before, but it's always nice to follow the [boy scout rule](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html). Can you, please, remove it?

Reply:
Hi @amiskov, off course!